.github/workflows/tests_gpu_python.yml

```
@@ -219,7 +229,7 @@ jobs:
           PL_BACKEND=${DEVICENAME} python scripts/configure_pyproject_toml.py
           python -m build
           python -m pip install dist/*.whl --force-reinstall --no-deps
-          PL_DEVICE=${DEVICENAME} python -m pytest tests/test_device.py $COVERAGE_FLAGS
+          PL_DEVICE=${DEVICENAME} python -m pytest tests/test_device.py $COVERAGE_FLAGS"
```

```suggestion
          PL_DEVICE=${DEVICENAME} python -m pytest tests/test_device.py $COVERAGE_FLAGS
```
I guess this `"` is not matching any?

---

This is weird. Thank you very much! :slightly_smiling_face: